src/pyne/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 message("")
 
-set(SRC_FILES pyne.cpp)
-set(PUB_HEADERS pyne.h)
+execute_process(COMMAND amalgamate_pyne.sh)
```

Comment:
Based on the failing tests, it seems that this may still not be run correctly???

Reply:
the problem is deeper that that.

basically the process if run is parallel, so the file are not there when needed to configure...

Reply:
Hrmm yeah I think we'll want to use the `add_custom_command`/`add_custom_target` commands here so we can add this step as a dependency of the PyNE library build.

Reply:
from what I read those are run at build time ...

Reply:
If we add a result code, can this force it to not be parallel?
https://stackoverflow.com/questions/52076948/check-exit-status-of-execute-process-in-cmake